spec/cool-tool_spec.rb

```diff
@@ -1,12 +1,13 @@
 require_relative '../cool-tool.rb'
 
 describe 'Cool Score' do
-  subject { CoolScore.new }
+  subject { CoolScore.new("Bob",30) }
```

Comment:
It's good practice to have a space between the two arguments here.


```suggestion
  subject { CoolScore.new("Bob", 30) }
```